platform/viewer/src/components/Header/Header.js

```diff
@@ -105,7 +101,11 @@ function Header(props) {
 }
 
 Header.propTypes = {
-  home: PropTypes.bool.isRequired,
+  // Study list, /
+  linkText: PropTypes.string,
+  linkPath: PropTypes.string,
+  useLargeLogo: PropTypes.bool,
```

Comment:
`home` prop to props that are more flexible and less context specific.
Use of app config is determined one layer higher. Helps for a dumber component.